static/js/settings/indexers.js

```diff
@@ -119,6 +119,19 @@ function _get_settings(){
         settings['Torrent'][checkbox.id] = is_checked(checkbox);
     });
 
+// INDEXERS['PRIVATETORRENT']
+    var required_fields = [];
+
+    var privateTrackers = {};
+
+    var danishbits = {};
+    danishbits.enabled = is_checked(document.querySelector("i[data-id='danishbits']"));
+    danishbits.username = document.querySelector("input[data-id='danishbits-username']").value;
+    danishbits.passkey = document.querySelector("input[data-id='danishbits-passkey']").value;
+    privateTrackers['danishbits'] = danishbits;
+
+    settings['PrivateTorrent'] = privateTrackers;
+
     if(blanks == true){
```

Comment:
This can probably be refactored to be a bit more flexible. Ideally we should be able to add a new private tracker without having to touch the javascript.

It might take a few changes to the markup, but you can probably make sure each tracker has a container that is a direct child of the form element, then loop over each of these, using data attributes in the container element to hold the key for the object, then find all children that are inputs and append their value to the settings object. The extra few milliseconds this takes is worth keeping everything flexible.